pages/account/statistics.js

```diff
@@ -155,7 +155,7 @@ export default function Statistics({ data, profile, progress }) {
         </div>
 
         <h1 className="text-4xl mb-4 font-bold">
-          Your Statistics for {profile.name} ({profile.username})
+          Your Statistics for {profile.name} ({profile.username}) - <u> <a href={"https://linkfree.io/" + (profile.username)}>View Page</a> </u>
```

Comment:
Thank you for your contribution

- I am not sure the `u` tag is the correct one to use
- It might be better to make the username clickable
- we should not hardcode the url, as it will change depending on the environment

Reply:
Great thanks 👍 

- Btw we should use `Link` not `a`.
- The current url can be found from the environment variables `process.env.NEXT_PUBLIC_BASE_URL`